### PR TITLE
Feature/video calling all page

### PR DESCRIPTION
**Pull request by dhruv-divtech**

### CODE REVIEW

File: src/views/protectedViews/videoCalling/NotPickedModel/NotePickedModel.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    justifyContent: 'center',
+    alignItems: 'center',
+    gap: theme.spacing(6)
+  }
+}));
+
+export const DialogContentMain = styled(Dialog)(({ theme }) => ({
+  '& .MuiDialog-paper': {
+    backgroundColor: '#07030E',
+    borderRadius: '12px',
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/videoCalling/NotPickedModel/NotePickedModel.styled.tsx:27]: use theme.spacing

File: src/views/protectedViews/videoCalling/RingingModel/RingingModel.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  flexDirection: 'column'
+}));
+
+export const DialogContentMain = styled(Dialog)(({ theme }) => ({
+  '& .MuiDialog-paper': {
+    backgroundColor: '#07030E',
+    borderRadius: '12px',
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/videoCalling/RingingModel/RingingModel.styled.tsx:37]: use theme.spacing

File: src/views/protectedViews/videoCalling/VideoCallEnded/VideoCallEnded.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    justifyContent: 'center',
+    alignItems: 'center',
+    gap: theme.spacing(6)
+  }
+}));
+
+export const DialogContentMain = styled(Dialog)(({ theme }) => ({
+  '& .MuiDialog-paper': {
+    backgroundColor: '#07030E',
+    borderRadius: '12px',
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/videoCalling/VideoCallEnded/VideoCallEnded.styled.tsx:27]: use theme.spacing

File: src/views/protectedViews/videoCalling/VideoCallEnded/index.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                      <FormattedMessage id="CallAgian" />
+                    </UINewTypography>
+                  </UIThemeShadowButton>
+                </FourBoxContent>
+              </ThirdBoxContent>
+              <UINewTypography variant="body" color="white.main">
+                <FormattedMessage id="ExploreOtherModels" />
+              </UINewTypography>
+            </FirstBoxContent>
+            <Box sx={{ display: 'flex', flexDirection: 'column', gap: 0.5 }}>
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/videoCalling/VideoCallEnded/index.tsx:72]: make style component

File: src/views/protectedViews/videoCalling/commonComponent/videoCallingCard.styled.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  alignItems: 'center',
+  justifyContent: 'center'
+}));
+
+export const ImgBoxContainer = styled('img')(({ theme }) => ({
+  width: '100%',
+  maxWidth: '80px',
+  height: '100%',
+  maxHeight: '80px',
+  borderRadius: '12px'
```
Comment (WorkspaceChetan) [inline src/views/protectedViews/videoCalling/commonComponent/videoCallingCard.styled.tsx:33]: use theme.spacing

Comment (dhruv-divtech) [reply src/views/protectedViews/videoCalling/commonComponent/videoCallingCard.styled.tsx:33]: done

File: src/views/protectedViews/videoCalling/VideoCallEnded/index.tsx (hunk 4 above)
Comment (dhruv-divtech) [reply src/views/protectedViews/videoCalling/VideoCallEnded/index.tsx]: done

File: src/views/protectedViews/videoCalling/VideoCallEnded/VideoCallEnded.styled.tsx (hunk 3 above)
Comment (dhruv-divtech) [reply src/views/protectedViews/videoCalling/VideoCallEnded/VideoCallEnded.styled.tsx]: done

File: src/views/protectedViews/videoCalling/RingingModel/RingingModel.styled.tsx (hunk 2 above)
Comment (dhruv-divtech) [reply src/views/protectedViews/videoCalling/RingingModel/RingingModel.styled.tsx]: done

File: src/views/protectedViews/videoCalling/VideoCallEnded/index.tsx (hunk 4 above)
Comment (dhruv-divtech) [reply src/views/protectedViews/videoCalling/VideoCallEnded/index.tsx]: done

File: src/views/protectedViews/videoCalling/commonComponent/videoCallingCard.styled.tsx (hunk 5 above)
Comment (dhruv-divtech) [reply src/views/protectedViews/videoCalling/commonComponent/videoCallingCard.styled.tsx:33]: done

File: src/views/protectedViews/videoCalling/AnotherCallModel/Another.styled.tsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    justifyContent: 'center',
+    alignItems: 'center',
+    gap: theme.spacing(6)
+  }
+}));
+
+export const DialogContentMain = styled(Dialog)(({ theme }) => ({
+  '& .MuiDialog-paper': {
+    backgroundColor: '#07030E',
+    borderRadius: '12px',
```
[the remark this answers is not in the document]
Comment (dhruv-divtech) [reply src/views/protectedViews/videoCalling/AnotherCallModel/Another.styled.tsx]: done

**Pull request merged**